topo_processor/util/configuration.py

```diff
@@ -0,0 +1,10 @@
+import os
+from tempfile import mkdtemp
+
+from dotenv import dotenv_values
+
+configuration = dotenv_values(".env")
+
+lds_cache_bucket: str = configuration["LDS_CACHE_BUCKET"]
+aws_role_config_path = os.path.expanduser(configuration["AWS_ROLES_CONFIG"])
+temp_folder: str = mkdtemp()
```

Comment:
This will be instanced on runtime but I don't know how to handle it without having to pass it as argument is all the call stack from `validate`